forch/forchestrator.py

```diff
@@ -128,6 +130,8 @@ def initialize(self):
         self._process_static_device_behavior()
         if self._faucetizer:
             self._faucetizer.flush_behavioral_config(force=True)
+        self._proxy = ForchProxy(self._config.proxy_server)
```

Comment:
ForchProxy should only be instantiated/started if specifically enabled. So, you could either trigger it off of the port being defied and do away with DEFAULT_PROXY_PORT as a default, or trigger it off of the proxy config block being defined.  Either way, an empty config file should not instantiate the proxy at all.

Reply:
Addressed